sdk/containerinstance/Azure.ResourceManager.ContainerInstance/CHANGELOG.md

```diff
@@ -1,29 +1,29 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0-beta.1 (2024-10-24)
 
 ### Features Added
 
+- Upgraded api-version tag from 'package-2023-05' to 'package-preview-2024-05'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/10b3107f2eafcd428a208ec5858f3ca9261b1123/specification/containerinstance/resource-manager/readme.md.
```

Comment:
did not have the tag detail in previous releases before, is this required later on?